core/common/src/test/java/alluxio/AlluxioTestDirectory.java

```diff
@@ -70,29 +66,18 @@ public static File createTemporaryDirectory(File parent, String prefix) {
    * @return the testing directory
    */
   private static File createTestingDirectory() {
-    final File tmpDir = new File(System.getProperty("java.io.tmpdir"), "alluxio-tests");
-    if (tmpDir.exists()) {
-      cleanUpOldFiles(tmpDir);
-    }
+    final File tmpDir = new File(System.getProperty("java.io.tmpdir"),
+        "alluxio-tests" + "-" + UUID.randomUUID());
     if (!tmpDir.exists()) {
```

Comment:
should this go directly to mkdir without checking for existence since it should never exist?

then this effectively is the same as `createTemporaryDirectory()` except that there's no prefix or `"-"` to prepend to the UUID, assuming the shutdown hook remains.

Reply:
the difference here is each testing process will have a unique root alluxio-tests root dir, they won't race to create it and fail some times.